java/src/jmri/jmrit/operations/locations/tools/ImportLocations.java

```diff
@@ -290,18 +294,16 @@ public void run() {
                 }
             }
 
-            // TODO import fields 14 through 22
+            // TODO import fields 15 through 23
 
-            if (inputLine.length >= FIELD_IGNORE_MINIMUM) {
-                String ignoreMin = inputLine[FIELD_IGNORE_MINIMUM].trim();
-                if (ignoreMin.length() > 0) {
-                    log.debug("setting the ignore minimum to {}", ignoreMin);
-                    Integer ignoreValue = null;
+            if (inputLine.length >= FIELD_TRACK_MINIMUM_POOL) {
+                String minPool = inputLine[FIELD_TRACK_MINIMUM_POOL].trim();
+                if (minPool.length() > 0) {
+                    log.debug("setting track pool minimum: {}", minPool);
                     try {
-                        ignoreValue = Integer.valueOf(ignoreMin);
-                        thisTrack.setBlockingOrder(ignoreValue);
+                        thisTrack.setMinimumLength(Integer.valueOf(minPool));
```

Comment:
Use Integer.parseInt() instead to solve the static analysis error.

See: https://stackoverflow.com/questions/32516893/findbugs-issue-with-boxing-unboxing-to-parse-a-primitive-with-integer-valueof